_includes/releases/v22.2/v22.2.1.md

```diff
@@ -0,0 +1,386 @@
+- TTL decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty print output. [#90723][#90723]
+- When running Cockroach inside of a Docker container on macOS, when mounting a host filesystem into the container, the total available capacity calculation of the filesystem could be reported incorrectly. This is now fixed. [#90873][#90873]
+- Fixed a bug that caused incorrect results and internal errors when a `LEFT JOIN` operated on a table with [virtual computed columns](../v22.2/computed-columns.html). The bug only presented when the optimizer planned a "paired joiner". Only values of the virtual columns would be incorrect—they could be be `NULL` when their correct value was not `NULL`. An internal error would occur in the same situation if the virtual column had a `NOT NULL` constraint. This bug was present since version v22.1.0. [#90997][#90997]
+- In large, [multi-region clusters](../v22.2/multi-region-overview.html), it was possible for the leasing mechanism used for jobs to get caught in a live-lock scenario whereby jobs could not be adopted. This bug has been resolved. [#91037][#91037]
+- `REASSIGN OWNED BY` could run into errors, if any descriptor owned by the user is currently being dropped. This is now fixed [#90388][#90388]
+- Fixed a bug that could cause `SELECT *` operations on tables with [virtual computed columns](../v22.2/computed-columns.html) undergoing schema changes to potentially fail. [#91007][#91007]
+- Fixed a bug where point lookups on the `pg_catalog.pg_type` table would fail to find the implicit record type that gets created for tables in the `pg_catalog`, `information_schema`, and `crdb_internal` schemas. [#91217][#91217]
+- Fixed a bug that prevented the usage of implicit record types for tables in the pg_catalog, information_schema, and crdb_internal schemas. [#91217][#91217]
+- Fixed a bug which caused a migration in v22.1 to fail to drop an index on `system.statement_diagnostics_requests` and, in turn, caused upgrades from v22.1 to v22.2, which had used the previous, faulty upgrade migration to now fail to create a new index with the same name as the index was thought to have been dropped. [#91308][#91308]
+- A nil pointer crash that could be hit when interleaving `SELECT FOR UPDATE SKIP LOCKED` statements has been resolved. [#91256][#91256]
+- Fixed a bug present only in earlier v22.2 release candidates, in which an [`ALTER PRIMARY KEY USING COLUMNS (x, x)`](../v22.2/alter-primary-key.html) statement would result in an internal error instead of the expected user-facing error with a pg-code. [#91478][#91478]
```

Comment:
nit: delete `earlier` (this is a minor release, so all `rc`s are earlier, and this does not appear designed to clarify between `rc`s. If it is, like "only in `rc.1` but not `rc.4`", then we need engineering to be clearer about this.)